DEVELOPMENT.md

```diff
@@ -158,20 +119,18 @@ Currently we need to follow this process:
 1. Change the contents of version.rb with the new version.
 2. Run `bundle install` with all Gemfiles to update all lockfiles.
 3. Run `rake license_finder:report:xml > licenses.xml`.
-4. (maybe) If you will generate a new CI image with `make ci-build`, change the
-   configuration of `.circleci/config.yml` to point to the future image.
-5. Modify CHANGELOG.md filling up data about notable changes along with
+4. Modify CHANGELOG.md filling up data about notable changes along with
    associated issue or PR numbers where available.
-6. Run task release:changelog:link_prs to link the issue or PR numbers in the
+5. Run task release:changelog:link_prs to link the issue or PR numbers in the
    CHANGELOG.md file and check the diff makes sense.
-7. Review and commit "apisonator: release X.Y.Z".
-8. Verify the tests pass and fix any issue that crops up. If at all possible,
+6. Review and commit "apisonator: release X.Y.Z".
+7. Verify the tests pass and fix any issue that crops up. If at all possible,
    deploy the version as a pre-release in a staging environment for further
    testing and early detection of issues.
-9. If you did step 4, build the new CI image, rebuild the dev image based on it,
+8. If you did step 4, build the new CI image, rebuild the dev image based on it,
```

Comment:
Is this step still needed?

Reply:
Not really. And I'm not sure what to do with the steps above mentioning the CHANGELOG